### PR TITLE
Update CONTRIBUTING.md

### LINKED ISSUE #201: Update CONTRIBUTING.md
- Contribution pathways

### PR DESCRIPTION
**Pull request by acocac**
### Summary

Fix #201

### List of changes proposed in this PR (pull-request)

* Make a more accessible CONTRIBUTING.md based on the Turing Way.

### What should a reviewer concentrate their feedback on?

- [ ] Do sections covered most of existing ways to contribute?
- [ ] All ok with the hyperlinks?
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,55 +1,260 @@
...
-## Report Bugs
+:green_heart: _We're excited that you're here and want to contribute._ :green_heart:
 
-Report bugs using GitHub issues.
+We want to ensure that every user and contributor feels welcome, included and supported to participate in the _EDS book_ community.
+We hope that the information provided in this document will make it as easy as possible for you to get involved.
 
-If you are reporting a bug, please include:
+We welcome all contributions to this project via GitHub issues and pull requests.
+Please follow these guidelines to make sure your contributions can be easily integrated into the projects.
```
Comment (annefou) [inline CONTRIBUTING.md]:
```suggestion
Please follow these guidelines to make sure your contributions can be easily integrated into the project.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,55 +1,260 @@
...
+2. :black_nib: Documentation Improvement ([preview here](https://github.com/alan-turing-institute/environmental-ds-book/issues/new?assignees=&labels=documentation&projects=&template=documentation-request.md&title=%5BDOC%5D)): This issue is to help us improve the documentation in the book or notebook repositories hosted in the [eds-book-gallery](https://github.com/eds-book-gallery) organisation. Alternatively you can just open a pull request with the suggested improvement.
+3. :green_book: Notebook Idea ([preview here](https://github.com/alan-turing-institute/environmental-ds-book/issues/new?assignees=&labels=documentation&projects=&template=documentation-request.md&title=%5BDOC%5D)): Use this template for submitting a notebook idea. You can find further info of the scope of notebooks ideas within the [Publishing guidelines](book/publishing/guidelines.md).
+4. :mag: PRE-REVIEW ([preview here](https://github.com/alan-turing-institute/environmental-ds-book/issues/new?assignees=&labels=prereview&projects=&template=prereview-template.md&title=%5BPRE+REVIEW%5D)): Editors-in-chiefs use this template for open a [PRE-REVIEW issue](book/publishing/guidelines/guidelines-eic.md#pre-review).
```
Comment (annefou) [inline CONTRIBUTING.md]:
```suggestion
4. :mag: PRE-REVIEW ([preview here](https://github.com/alan-turing-institute/environmental-ds-book/issues/new?assignees=&labels=prereview&projects=&template=prereview-template.md&title=%5BPRE+REVIEW%5D)): Editors-in-chiefs use this template to open a [PRE-REVIEW issue](book/publishing/guidelines/guidelines-eic.md#pre-review).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,55 +1,260 @@
...
+5. :eyes: REVIEW ([preview here](https://github.com/alan-turing-institute/environmental-ds-book/issues/new?assignees=&labels=review&projects=&template=review-template.md&title=%5BREVIEW%5D)): Editors-in-chiefs use this template for open a [REVIEW issue](book/publishing/guidelines/guidelines-eic.md#review).
```
Comment (annefou) [inline CONTRIBUTING.md]:
```suggestion
5. :eyes: REVIEW ([preview here](https://github.com/alan-turing-institute/environmental-ds-book/issues/new?assignees=&labels=review&projects=&template=review-template.md&title=%5BREVIEW%5D)): Editors-in-chiefs use this template to open a [REVIEW issue](book/publishing/guidelines/guidelines-eic.md#review).
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,55 +1,260 @@
...
+
+   ```
+   cd ./book/website
+   jupyter-book build .
+   ```
+
+## Representing _EDS book_
+
+We would LOVE people to give talks about the project or represent _The Turing Way_ in other ways!
+We have created a [promotion pack](https://github.com/alan-turing-institute/environmental-ds-book/tree/master/communications/promotion-pack) for you to reuse.
```
Comment (annefou) [inline CONTRIBUTING.md]: Do we mean _The Turing Way_ or _The EDS Book_ ?

**Review by annefou — APPROVED** [review]
Thanks for this nice and comprehensive guidelines.

File: CONTRIBUTING.md (hunk 1 above)
Comment (acocac) [reply CONTRIBUTING.md]: Good catch.

File: CONTRIBUTING.md (hunk 4 above)
Comment (acocac) [reply CONTRIBUTING.md]: You're right. I should be _EDS book_

Comment (acocac) [reply CONTRIBUTING.md]: Fixed in commit [659ba3](https://github.com/alan-turing-institute/environmental-ds-book/pull/219/commits/659ba3b4c94daa3e007095c0bca8c30a6189f2f6)

**Comment by acocac** [comment]
> Thanks for this nice and comprehensive guidelines.

Thank you! Let's revisit CONTRIBUTING.md at later stage. I think we should add a short section how to contribute to new notebooks or cross-reference guidelines if necessary.

**Pull request merged**